source/_static/custom.css

```diff
@@ -2,57 +2,116 @@
   --image-width: 25em;
 }
 
-.wy-nav-top, .wy-nav-side, .wy-side-nav-search {
-    background: #4e4a4a;
+h1 {
+  font-weight: 500;
+}
+
+h2,
+h3 {
+  font-weight: 500;
+
+  margin-bottom: 0;
+}
+
+li img {
+  padding: 1rem;
+  margin-bottom: 2rem;
+}
+
+.admonition {
+  margin: 2rem 1rem;
+
+  border-top: 1px solid #e5e7eb;
+  border-bottom: 1px solid #e5e7eb;
+  border-right: 1px solid #e5e7eb;
```

Comment:
These colors should be dependent on a prefers-color-scheme media query, the light border on a dark background looks really weird